### DISCUSSION TITLE
What is the support model of Bellhop?

### DISCUSSION THREAD

**Discussion by tomkerkhove** [Azure/bellhop#86] [Q&A] [answered]
While working on [Azure Autoscaling Toolbox](http://azure-autoscaling-toolbox.cloud/), I wanted to ensure I got this right - Bellhop is using community-based support and is not available through Azure support, is that correct?

**Comment by CloudViking** [comment, marked as answer]
Hi @tomkerkhove!

Sorry for the delay... You are correct, Bellhop is using a 100% community-based support model. @NillsF, @DCMattyG, and I try to keep up as much as we can, but with the new FY being so busy I don't think any of us have had the time we'd hoped to invest here. 

Azure Autoscaling Toolbox looks great, by the way!

**Reply by tomkerkhove** [reply]
No problem at all, I understand! I just wanted to make sure I got https://azure-autoscaling-toolbox.cloud//autoscalers/bellhop.html right but open to any PR!
